README.md

```diff
@@ -11,12 +11,22 @@ See [examples](examples/) for how to use the PI.
 
 - libjudy-dev
 - libreadline-dev
+- libtool-bin
```

Comment:
These should probably be listed as `make check` dependencies. They are not needed except to run the full test suite when configure is run with `--with-bmv2`

Reply:
yes, and I fix it, could you please review again? thanks!